source/partials/_head_big.html.slim

```diff
@@ -5,8 +5,7 @@
         img.hero__logo alt="Kiwi Ruby - New Zealand Ruby Conference 2017" src="./images/kiwiruby.svg"
       .hero__slogan
         span.hero__hidden Kiwi Ruby
-        br
         p.hero__date
           ' November 2-3 2017
-        p.hero__date
+          br
```

Comment:
Why this change? Seems unrelated

Reply:
@mermop It is unrelated, yes! While I was there I thought to squish the hero date and location together under the logo on desktop, the line spacing between them seemed too high.